web/app.js

```diff
@@ -254,6 +258,24 @@ var PDFViewerApplication = {
     ]).catch(function (reason) { });
   },
 
+  _initializeL10n() {
+    if ((typeof PDFJSDev === 'undefined' ||
+        !PDFJSDev.test('FIREFOX || MOZCENTRAL')) &&
+        PDFViewerApplication.viewerPrefs['pdfBugEnabled']) {
```

Comment:
The above condition isn't exactly equal to the current ones, since in `master` it's possible to change the `#locale` in dev-mode (i.e. when using `gulp server`) without having the 'pdfBugEnabled' pref set.
I think it'd be nice if we could keep that behaviour, so this needs to look something like:
```javascript
if (((typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) ||
     this.viewerPrefs['pdfBugEnabled']) &&
    (typeof PDFJSDev === 'undefined' ||
     !PDFJSDev.test('FIREFOX || MOZCENTRAL'))) {
```

Obviously that isn't as readable as one would like, but it may be possible to simplify the condition somewhat.